.gitignore

```diff
@@ -0,0 +1,17 @@
+# Ignore everything
+*
```

Comment:
If we are starting from the scratch I would suggest to revert this to a standard "way"... this "ignore everything" pattern resulted several times in not adding important files... ;)